src/encoded/types/gene.py

```diff
@@ -0,0 +1,30 @@
+from snovault import (
+    calculated_property,
+    collection,
+    load_schema,
+)
+from .base import (
+    SharedItem,
+)
+
+
+@collection(
+    name='genes',
+    unique_key='gene:geneid',
+    properties={
+        'title': 'Genes',
+        'description': 'Listing of genes',
+    })
+class Gene(SharedItem):
+    item_type = 'gene'
+    schema = load_schema('encoded:schemas/gene.json')
+    name_key = "geneid"
+    embedded = ['organism']
+
+    @calculated_property(schema={
+        "title": "Title",
+        "type": "string",
+    })
+    def title(self, request, organism, symbol):
+        organism_props = request.embed(organism, '@@object')
+        return u'{} ({})'.format(symbol, organism_props['scientific_name'])
```

Comment:
Did you consider using organism_props.get('scientific_name')?  In this case the function would return 
```
'whatever-symbol (None)' 
```
when the key doesn't exist.  The current implementation would fail if for some reason the key wasn't there.  But that could be a desired result?

Reply:
This should probably never happen as the loading script should not be loading any gene objects for organisms that aren't already registered. We should probably be quite clear in the schema/docs as to which organisms are supported.

However, organism.scientific_name isn't required and it probably should be. We should address that in a separate ticket.